experimental/CHANGELOG.md

```diff
@@ -17,7 +17,8 @@ All notable changes to experimental packages in this project will be documented
     * (internal) OTLPExporterBrowserBase: `RequestType` has been replaced by a `ResponseType` type-argument
     * (internal) OTLPExporterNodeBase: `ServiceRequest` has been replaced by a `ServiceResponse` type-argument
     * (internal) the `@opentelemetry/otlp-exporter-proto-base` package has been removed, and will from now on be deprecated in `npm`
-* fix(instrumentation)!: remove unused supportedVersions from Instrumentation interface [#4694](https://github.com/open-telemetry/opentelemetry-js/pull/4694) @blumamir
+    * feat(instrumentation): add default value for config in base instrumentation constuctor [#4695](https://github.com/open-telemetry/opentelemetry-js/pull/4695): @blumamir
+    * fix(instrumentation)!: remove unused supportedVersions from Instrumentation interface [#4694](https://github.com/open-telemetry/opentelemetry-js/pull/4694) @blumamir
```

Comment:
These entries should be dedented, right?

Also, I'm slightly confused, the PR title says "remove default value ...", but this entry says "add default value ..."

Reply:
Yes I committed @dyladan change suggestion [here](https://github.com/open-telemetry/opentelemetry-js/pull/4695#discussion_r1599033711). honestly, I don't mind what will be the exact text. will align it to the PR title.